pkgs/by-name/tm/tmc-cli/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "tmc-cli";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
+    owner = "rage";
+    repo = "tmc-cli-rust";
+    rev = "v1.1.2";
```

Like this:
```suggestion
    tag = "v${finalAttrs.version}";
```
(also use `tag` if you need a tag)